internal/pkg/kernel/kernel.go

```diff
@@ -72,28 +72,34 @@ func ListKernels() ([]string, error) {
 }
 
 func Build(kernelVersion string, root string) (string, error) {
+
 	kernelImage := path.Join(root, "/boot/vmlinuz-"+kernelVersion)
 	kernelDrivers := path.Join(root, "/lib/modules/"+kernelVersion)
 	kernelDestination := KernelImage(kernelVersion)
 	driversDestination := KmodsImage(kernelVersion)
 
 	// Create the destination paths just in case it doesn't exist
-	err := os.MkdirAll(path.Dir(kernelDestination), 0755)
-	if err != nil {
-		return "", fmt.Errorf("failed to create kernal dest: %s", err)
-	}
-
-	err = os.MkdirAll(path.Dir(driversDestination), 0755)
-	if err != nil {
-		return "", fmt.Errorf("failed to create driver dest: %s", err)
-	}
+  if  err := os.MkdirAll(path.Dir(kernelDestination), 0755) ; err != nil {
+    return "", err
+  }
+	if err := os.MkdirAll(path.Dir(driversDestination), 0755) ; err != nil {
+    return "", err
+  }
 
 	if !util.IsFile(kernelImage) {
-		return "", errors.New("Could not locate kernel image")
+		// allow uncompressed kernels (can be handy for aarch64)
+		kernelImage = path.Join(root, "/boot/vmlinux-"+kernelVersion)
+		if !util.IsFile(kernelImage) {
+			return "", errors.New("Could not locate kernel image")
```

Comment:
This is okay for now, but for the future error message should not contain caps, or newlines.